### DISCUSSION TITLE
Default settings when upgrading packages

### DISCUSSION THREAD

**Discussion by AnonymousWP** [microsoft/winget-cli#1953] [General]
Apologies if it has been asked already or if this is the wrong place to ask, but couldn't really find a straightforward answer on the repo nor on Google.

So what are the default settings when I'm upgrading an existing package? For example, when I installed Python the first time, I could check the box that adds Python to PATH, I did not create a start menu shortcut but I did check the box for creating a desktop shortcut. How does Winget handle this? Does it simply check all boxes by default or does it use some configuration file that every application generates after installing? I have found [this](https://www.techwatching.dev/posts/winget-override#:~:text=winget%20install%20official%20documentation%20says,some%20parameters%20for%20the%20program.), but this only mentions how to override and not what the defaults are that Winget uses. Plus, we'd have to look up documentation per application I guess. Hopefully it will be possible to have one repository (or built-in Winget) with all available override options per application.

Besides, is anyone aware of a feature request about changelogs (equivalent to `apt listchanges`)? See: https://manpages.debian.org/testing/apt-listchanges/apt-listchanges.1.en.html.

**Comment by denelon** [comment]
Related to:

* #607
